src/meili/match_route.cc

```diff
@@ -54,12 +54,13 @@ namespace meili {
 EdgeSegment::EdgeSegment(baldr::GraphId the_edgeid,
                          float the_source,
                          float the_target,
+                         int the_restriction_idx,
```

Comment:
putting this here seems dangerous for existing calls. would putting it on the end be easier?